### DISCUSSION TITLE
Is it possible to combine multiple tailwind.config.js files?

### DISCUSSION THREAD

**Discussion by webbie** [tailwindlabs/tailwindcss#3107] [Help]
Hello, I have a situation where I need to generate css for 3 different themes (separate files, 1 css for each theme).
`tailwind.config.js` configurations for each theme are almost identical, the only differences are in colors. 
Is it possible to have 3 configuration files for each theme, where they'll include main configuration file?

**Comment by MrX8503** [comment]
~Is this what you're looking for?~ Answer here. https://github.com/tailwindlabs/tailwindcss/discussions/3107#discussioncomment-212101

```js
const defaultConfig = require('path-to-default-config');

// theme-1-tailwind.config.js
module.exports = {
    ...defaultConfig,
    theme: {
        ...defaultConfig.theme,
        colors: {
            ...defaultConfig.theme.colors,
            // add colors here
        }
    }
};

// theme-2-tailwind.config.js
...

// theme-3.tailwind.config.js
....
```

**Reply by sujayk46** [reply]
hiii can u just brieft this further

**Reply by sujayk46** [reply]
like how to exactly switcg between 2 themes

**Comment by simonswiss** [comment]
Hello!

Not 100% what your case scenario is, but sounds like [presets](https://tailwindcss.com/docs/presets) is what you're looking for!

**Reply by webbie** [reply]
That is it! Thanks!

**Comment by cognivore** [comment]
Please mark as answered. Presets do the job well.

**Reply by quantuminformation** [reply]
nice work guys
